docs/src/main/asciidoc/redis-dev-services.adoc

```diff
@@ -0,0 +1,37 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Dev Services for Redis
+
+:extension-status: preview
+include::./attributes.adoc[]
+
+Quarkus supports a feature called DevServices that allows you to create various datasources without any config.
```

Comment:
Shouldn't this just go in the main redis doc as a subsection?

Reply:
Kafka and AMQP already use this structure. The getting started contains admonitions referring to this file. Dev service configuration is more reference content than getting started content.